### PR TITLE
Fix for Commonjs transforming issue for 'return' outside of function

### LINKED ISSUE #11: Commonjs transforming issue for 'return' outside of function
I am getting following error 

Error transforming /home/travis/build/GeekAb/picker.js/node_modules/beeper/index.js with 'commonjs' plugin: 'return' outside of function (9:1) in /home/travis/build/GeekAb/picker.js/node_modules/beeper/index.js
[08:58:22] 'rollup:iife' errored after 5.63 s
[08:58:22] SyntaxError: Error transforming /home/travis/build/GeekAb/picker.js/node_modules/beeper/index.js with 'commonjs' plugin: 'return' outside of function (9:1) in /home/travis/build/GeekAb/picker.js/node_modules/beeper/index.js

### PR DESCRIPTION
**Pull request by GeekAb**
Fixes #11

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-	return;
-}
+(function() {
+	if (!process.stdout.isTTY ||
+        process.argv.indexOf('--no-beep') !== -1 ||
+        process.argv.indexOf('--beep=false') !== -1) {
+        module.exports = function () {};
+        return;
+    }
+})
```
Comment (kevva) [inline index.js]: I would put the `if` statement on top in the `module.exports` further down instead and skip this wrapping.

Comment (GeekAb) [inline index.js]: That will also work. I was just following the current structure.

Comment (GeekAb) [inline index.js]: Just updated the request.

Comment (GeekAb) [inline index.js]: Hi Guys , 

can someone please take a look at this?

Comment (sindresorhus) [inline index.js]: @GeekAb No need to ping right away. We get notified when the PR is updated. It's also not correct. Please look at the code more closely.

File: index.js [hunk 2]
```diff
@@ -29,6 +22,13 @@ function melodicalBeep(val, cb) {
 }
 
 module.exports = function (val, cb) {
+    
+    if (!process.stdout.isTTY ||
+        process.argv.indexOf('--no-beep') !== -1 ||
+        process.argv.indexOf('--beep=false') !== -1) {
+        return;
+    }
+
```
Comment (sindresorhus) [inline index.js]: Wrong indentation and you also left some trailing whitespace in there, but I'll fix when merging.

**Pull request merged**